packages/nameguard-js/src/nameguard-js.test.ts

```diff
@@ -0,0 +1,70 @@
+import { describe, it, expect } from "vitest";
+import { createPublicClient, http } from "viem";
+import { mainnet } from "viem/chains";
+import { createClient } from "./nameguard-js";
+
+const PROVIDER_URI_MAINNET = process.env.PROVIDER_URI_MAINNET;
+
+describe("NameGuardJS", () => {
+  it("should compute secure primary name", async () => {
+    const publicClient = createPublicClient({
+      chain: mainnet,
+      transport: http(PROVIDER_URI_MAINNET),
+    });
+
+    const localNameguard = createClient({
+      // not a real endpoint, will error if used
+      endpoint: "http://localhost:1234",
```

Comment:
I think we should be ok to just use the default NameGuard endpoint and the test should still pass?

Reply:
The test checks if the local provider was used, this endpoint enforces this.